package.json

```diff
@@ -20,7 +20,7 @@
 		]
 	},
 	"dependencies": {
-		"@nextui-org/react": "2.4.2",
+		"@nextui-org/react": "^2.4.2",
```

Comment:
you do not need to update the dependencies (as we are currently trying to resolve a BUG in the https://github.com/nextui-org/nextui/issues/3510)